orb-ui/src/sound/mod.rs

```diff
@@ -352,19 +441,18 @@ impl Fake {
         // Get a output stream handle to the default physical sound device
         let (stream_task, stream_handle) =
             StreamTask::new().wrap_err("failed to create stream task")?;
-        let sink = rodio::Sink::try_new(&stream_handle)?;
-        let (tx, mut rx) = mpsc::channel(1);
-
-        // TODO load config
-
+        let sink = Arc::new(Mutex::new(rodio::Sink::try_new(&stream_handle)?));
+        let (tx, mut rx) = mpsc::channel(5);
```

Comment:
magic number alert: I suggest making it a const and putting it at the top of the file. Also, why 5 and not 1 or 0, or a power of 2?